src/Types/Saber/Saber.cpp

```diff
@@ -105,8 +124,8 @@ namespace Qosmetics
         Transform* basicSaberModel = get_transform()->Find(modelManager->get_basicSaberModelName());
         if (basicSaberModel) SaberUtils::HideObjects(basicSaberModel->get_gameObject(), false, false);
         Il2CppString* saberName = (saberType == 0) ? modelManager->get_leftSaberName() : modelManager->get_rightSaberName();
-        Transform* saber = get_transform()->Find(saberName);
-        if (saber) saber->get_gameObject()->SetActive(false);
+        modelObject = get_transform()->Find(saberName);
+        if (saber) modelObject->get_gameObject()->SetActive(false);
```

Comment:
I think this if check has an identifier that doesn't exist now?

Reply:
this compiles because it is recognized as an enum

![image](https://user-images.githubusercontent.com/51795403/128766670-9a2e0c71-57b6-4ccb-9b09-ef2b1a00636d.png)